docs/annexes/annex-06-pid-rulebook.md

```diff
@@ -130,24 +132,50 @@ number in the domestic namespace.
 EXAMPLE: The first domestic PID namespace for Germany could be
 "eu.europa.ec.eudi.pid.de.1".
 
-PID Providers SHALL use the same domestic namespace for both ISO/IEC
-18013-5-compliant PIDs and SD-JWT-compliant PIDs, see section 2.5.
-
 A PID Provider that defines a domestic namespace SHALL publish the
 namespace, including all data element identifiers, their definition,
 presence and encoding format, on a website that is publicly available. A
 central registry for such domestic namespace definitions MAY be
 established in the future.
 
-### 2.3 PID attributes
+### 2.3 SD-JWT VC Type (SD-JWT VC format)
+SD-JWT VC defines the Verifiable Credential Type (`vct`). A
+type comes with associated metadata that, for instance, provides information about the type itself, outlines a schema detailing the claims that are optional or mandatory in the SD-JWT VC, and specifies their display methods. Additionally, a type can inherit from another type, enabling the creation of domestic types based on a broader EU-wide standard. The information regarding a type can be automatically discovered.
+
+The information provided in this document SHALL be specified as a
+Verifiable Credential Type in the format defined by SD-JWT VC, including
+a schema for PIDs, display (rendering) information, and other metadata
```

Comment:
```suggestion
a schema for PIDs, display information, and other metadata
```

Reply:
I like the explicit mention of rendering - we convery text and possible images for a credential